TI-reports/2024/2024-Q3-SCI-WG.md

```diff
@@ -0,0 +1,196 @@
+# 2024 Q3 Supply Chain Integrity (SCI) WG
+
+## **Overview**
+
+**Mission**: Scalable standardized attestable practices for supply chain security
+
+**Key Resources**
+
+
+
+*   [Vision Doc](https://docs.google.com/document/d/1SuJHaCr89Ih6TFvAIH2WM5M4_MlXwM8mqHynUBuungE/edit?resourcekey=0-wg-QROzZFa4Ju_uN_wOBNQ) and [Charter](https://github.com/ossf/wg-supply-chain-integrity/blob/main/governance/CHARTER.md)
+*   [GitHub](https://github.com/ossf/wg-supply-chain-integrity); 24 contributors (=)
+*   [Mailing List](https://lists.openssf.org/g/openssf-supply-chain-integrity); 181 members (**⇧3**)
+*   [Slack](https://app.slack.com/client/T019QHUBYQ3/C01A1MA7A1K); 839 members (**⇧94**)
+*   [Meeting Notes](ssci.io/sci-notes); 15 attendees average since last TAC briefing (**⇧4**)
+
+**tl;dr:** Strong membership and attendance trends, and the formal addition of Zarf and Security Insights to SCI WG. With the retirement of FRSCA we now have SLSA, S2C2F, GUAC, gittuf, Zarf, and Security Insights; with interest expressed from the Trusted Repository Security Initiative (TRSI) and Clean Dependency Project in addition.
+
+**We’re ready to graduate this WG when we have time for the paperwork.**
+
+**Latest news**:
+
+
+
+*   SLSA v1.1 readying for release (7 TODOs left)
+*   S2C2F working with SLSA on Dependency Track
+*   SLSA Source Track in draft
+*   Funding approved for gittuf GitHub App
+*   Zarf 1.0 anticipated by EoY
+
+
+## SLSA
+
+
+### Purpose
+
+A pragmatic supply chain security framework covering key functional areas, and providing for improved comprehension and security of software supply chains.
+
+
+### Current Status
+
+v1 (2023) focused on build integrity and provenance (“Build Track”).
+
+
+### Up Next
+
+v1.1 (soon!) with an updated threat model and minor clarifications
+
+Source Track in draft
+
+Hardware Attestations Track in development
```

Comment:
Nit: We've evolved this into the Attested Build Environments Track. The PR to enter it into draft is currently open and under review.

Reply:
Updated, thanks!